Additional LDFLAGS forgotten in first patch

Patch by Kevin McCarthy <signals@gentoo.org>

--- src/plug/hack/Makefile
+++ src/plug/hack/Makefile
@@ -30,19 +30,19 @@
 scriptedit.${SO}:
 else
 gtk-hello.${SO}:
-	-${CC} `pkg-config gtk+-2.0 --cflags --libs` gtk-hello.c -I ../.. -fPIC -shared -g -o gtk-hello.${SO}
+	-${CC} ${LDFLAGS} `pkg-config gtk+-2.0 --cflags --libs` gtk-hello.c -I ../.. -fPIC -shared -g -o gtk-hello.${SO}
 
 gtk-actions.${SO}:
-	-${CC} `pkg-config gtk+-2.0 --cflags --libs` gtk-actions.c -I ../.. -fPIC -shared -g -o gtk-actions.${SO}
+	-${CC} ${LDFLAGS} `pkg-config gtk+-2.0 --cflags --libs` gtk-actions.c -I ../.. -fPIC -shared -g -o gtk-actions.${SO}
 
 gtk-prefs.${SO}:
-	-${CC} `pkg-config gtk+-2.0 --cflags --libs` gtk-prefs.c -I ../.. -fPIC -shared -g -o gtk-prefs.${SO}
+	-${CC} ${LDFLAGS} `pkg-config gtk+-2.0 --cflags --libs` gtk-prefs.c -I ../.. -fPIC -shared -g -o gtk-prefs.${SO}
 
 gtk-topbar.${SO}:
-	-${CC} `pkg-config gtk+-2.0 --cflags --libs` gtk-topbar.c -I ../.. -fPIC -shared -g -o gtk-topbar.${SO}
+	-${CC} ${LDFLAGS} `pkg-config gtk+-2.0 --cflags --libs` gtk-topbar.c -I ../.. -fPIC -shared -g -o gtk-topbar.${SO}
 
 scriptedit.${SO}:
-	-${CC} `pkg-config gtk+-2.0 --cflags --libs` scriptedit.c -I ../.. -fPIC -shared -g -o scriptedit.${SO}
+	-${CC} ${LDFLAGS} `pkg-config gtk+-2.0 --cflags --libs` scriptedit.c -I ../.. -fPIC -shared -g -o scriptedit.${SO}
 endif
 
 python.${SO}:
--- src/rsc/gtk/Makefile
+++ src/rsc/gtk/Makefile
@@ -1,16 +1,16 @@
 all: Display SetRegister EntryDialog SeekTo
 
 Display:
-	${CC} `pkg-config --libs --cflags gtk+-2.0` Display.c -o Display
+	${CC} ${LDFLAGS} `pkg-config --libs --cflags gtk+-2.0` Display.c -o Display
 
 SetRegister:
-	${CC} `pkg-config --libs --cflags gtk+-2.0` SetRegister.c -o SetRegister
+	${CC} ${LDFLAGS} `pkg-config --libs --cflags gtk+-2.0` SetRegister.c -o SetRegister
 
 EntryDialog:
-	${CC} `pkg-config --libs --cflags gtk+-2.0` EntryDialog.c -o EntryDialog
+	${CC} ${LDFLAGS} `pkg-config --libs --cflags gtk+-2.0` EntryDialog.c -o EntryDialog
 
 SeekTo:
-	${CC} `pkg-config --libs --cflags gtk+-2.0` SeekTo.c -o SeekTo
+	${CC} ${LDFLAGS} `pkg-config --libs --cflags gtk+-2.0` SeekTo.c -o SeekTo
 
 gtkaml:
 	gtkamlc -C --pkg posix --pkg gtk+-2.0 Display.gtkaml
